benchmarks/graphics_pipeline/main.cpp

```diff
@@ -189,6 +194,13 @@ void ProjApp::Config(ppx::ApplicationSettings& settings)
     settings.grfx.swapchain.depthFormat = grfx::FORMAT_D32_FLOAT;
 }
 
+void ProjApp::InitKnobs()
+{
+    pKnobAlphaBlend = std::move(mKnobManager.CreateKnob<ppx::KnobCheckbox>("alpha_blend", false));
```

Comment:
You don't need to move return value.

Also, std::move is one of those thing that you should understand what it does before using it.

Reply:
That use is appropriate.

It actually does something that saves a few cpu cycle, although if you really want to save a few nano seconds of cpu cycle it's probably better to use `mKnobs.emplace_back(std::move(newKnob))`.